test/flycheck-test.el

```diff
@@ -3548,6 +3544,22 @@ Why not:
    '(4 2 error "Syntax error, maybe a missing semicolon?"
        :checker processing)))
 
+(flycheck-ert-def-checker-test pug pug syntax-error
+  (flycheck-ert-should-syntax-check
+   "language/pug/pug.pug" 'pug-mode
+   '(2 1 error "unexpected token \"indent\"" :checker pug)))
+
+(flycheck-ert-def-checker-test pug pug include-extends-error
+  (flycheck-ert-should-syntax-check
+   "language/pug/pug-extends.pug" 'pug-mode
+   '(1 nil error
+       "the \"basedir\" option is required to use includes and extends with \"absolute\" paths" :checker pug)))
```

Comment:
Please move the message to the line before, and instead break before `:checker`.  That's how we did it in the other tests, I find that it's easier to read this way.